webapp/src/containers/addOrViewComments/index.tsx

```diff
@@ -46,6 +46,12 @@ const AddOrViewComments = () => {
         resetFieldStates();
     }, []);
 
+    const getCommentsPayload = (): CommentsPayload => ({
+        record_type: pluginState.openCommentModalReducer.data?.recordType,
+        record_id: pluginState.openCommentModalReducer.data?.recordId,
+        comments,
+    });
```

Comment:
Please check if we need to add comments for all payloads
```suggestion
    const getCommentsPayload = (includeComments: boolean): CommentsPayload => ({
        record_type: pluginState.openCommentModalReducer.data?.recordType,
        record_id: pluginState.openCommentModalReducer.data?.recordId,
        ...(includeComments ? {comments} : {}),
    });
```

Reply:
I don't think we need to do this. We are sending the payload in `POST` request only in which we will need comments. In other requests (`GET`) we are using `record_type` and `record_id` in URL paths and comments are unused